.github/workflows/fosstars-report.yml

```diff
@@ -7,6 +7,9 @@ on:
 env:
   MVN_MULTI_THREADED_ARGS: --batch-mode --no-transfer-progress --fail-at-end --show-version --threads 1C
   JAVA_VERSION: 17
+  CVE_CACHE_KEY: cve-db
```

Comment:
moved from separate workflow to this workflow, because we only need the DB in this job, and unifying them is more reliable